### PR TITLE
Create WishList model.

### PR DESCRIPTION
**Pull request by ProkhvatylovArtem**
Closes issue #56.

### CODE REVIEW

File: ProjectShop/WishList/models.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+from django.db import models
+
+from UserApp.models import User
+
+
+class WishList(models.Model):
+    user_id = models.ForeignKey(User, on_delete=models.CASCADE)
+    # product_id = models.ForeignKey(Product, on_delete=models.CASCADE)
```
Comment (OlehHnyp) [inline ProjectShop/WishList/models.py:8]: We'd better use many to many relationship to Product, then we could include more than one product inside wishlist.

**Review by OlehHnyp — APPROVED** [review]

File: ProjectShop/WishList/models.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
+from django.db import models
+
+from UserApp.models import User
+
+
+class WishList(models.Model):
+    user_id = models.ForeignKey(User, on_delete=models.CASCADE)
+    # products = models.ManyToManyField(Product, related_name='wishlists')
```
Comment (v-lavrushko) [inline ProjectShop/WishList/models.py:8]: why this line is commented?

**Review by v-lavrushko — CHANGES_REQUESTED** [review]

File: ProjectShop/WishList/models.py (hunk 2 above)
Comment (ProkhvatylovArtem) [reply ProjectShop/WishList/models.py:8]: Because we haven't merged the product model to our project yet.

**Review by MykhailoGrynenko — APPROVED** [review]
LGTM

**Review by v-lavrushko — APPROVED** [review]

**Pull request merged**